### DISCUSSION TITLE
Datagrid cell or row tooltip

### DISCUSSION THREAD

**Discussion by wgraaf** [AvaloniaUI/Avalonia#11142] [Q&A] [answered]
How can I show a tooltip for a datagrid cell or row?
The tooltip text would be bound to a property of an item in the observable collection the grid is bound to.

**Comment by maxkatz6** [comment]
This style should work
```xaml
<Style Selector="DataGrid#MyDataGrid DataGridRow">
   <Setter Property="Tooltip.Tip" Value="{Binding TooltipText}" />
</Style>
```

Alternatively, you can use DataGrid.LoadedRow event and access Row control from the arguments.

**Comment by wgraaf** [comment, marked as answer]
<DataGrid.Styles>
              <Style Selector="DataGridRow">
                <Setter Property="ToolTip.Tip" Value="{Binding ToolTipText}" />
              </Style>
            </DataGrid.Styles>


did the trick. thnks!

**Reply by nbeck-SMT** [reply]
The data context for the tooltip seems to be the  parent view model. How can I access the row data?
